cli/src/semgrep/state.py

```diff
@@ -60,46 +60,6 @@ def is_scan_invocation() -> bool:
         command_name = ctx.command.name if hasattr(ctx, "command") else "unset"
         return command_name == "scan"
 
-    @staticmethod
-    def is_supply_chain() -> bool:
-        """
-        Returns True iff the current CLI invocation is a supply chain (SCA) invocation.
-        """
-        ctx = get_context()
-        command_name = ctx.command.name if hasattr(ctx, "command") else "unset"
-        is_scan = command_name == "scan"
-        params = ctx.params if hasattr(ctx, "params") else {}
-        # NOTE: For `semgrep scan`, supply-chain is passed via --config
```

Comment:
Yes, resolving cloud configurations (and local overrides) was out of scope for this PR. We only are applying the new CLI UX for the `scan` command, so handing the `ci` case is not needed at this point in time.

Reply:
Oh I see. Then please add a comment somewhere saying this is limited to semgrep scan, because ci does not need a nice display, but that we're not handling the --code and --secrets from semgrep ci.

Reply:
Done!